main/ninja/APKBUILD

```diff
@@ -39,6 +39,9 @@ package() {
 
 	install -m644 -D misc/bash-completion \
 		"$pkgdir/usr/share/bash-completion/completions/ninja"
+
+	install -m644 -D misc/ninja_syntax.py \
+		"$pkgdir/usr/lib/python3.6/site-packages/ninja_syntax.py"
```

Comment:
Please consider what happens when python3.7 is used.

We should not hard code the python version in the installation path.